doc/media.md

```diff
@@ -219,7 +222,7 @@ be called with each desired media type _before_ using `startMedia`. This will en
   function will stop and the error thrown (any successful media will be destroyed).
   This is is line with `getUserMedia`'s current behavior in the browser.
 
- If `retryOnFailure` is `true` (default), the SDK will have the following behavior:
+ If `mediaReqOptions.retryOnFailure` is `true` (default), the SDK will have the following behavior:
```

Comment:
NIT: We should use `IMediaRequestOptions.retryOnFailure` or `options.retryOnFailure` instead of assuming people know what we are talking about here.